### ISSUE TITLE
[BUG] Cannot run OpenShift Local - SSL handshake issues

### ISSUE THREAD

**Issue by bandrzejewski** [crc-org/crc#3363]
### General information

OS: macOS
Hypervisor: vfkit
Did you run crc setup before starting it (Yes/No)? Yes
Running CRC on: Laptop and Desktop

## CRC version
```bash
CRC version: 2.9.0+589ab2cd
OpenShift version: 4.11.3
Podman version: 4.2.0
```
  
## CRC status
```bash
CRC VM:          Running
OpenShift:       Unreachable (v4.11.3)
Podman:          
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     37.11GB
Cache Directory: /Users/user/.crc/cache
```

## CRC config
```bash
- consent-telemetry                     : no
- http-proxy                            : http://corporate.proxy:8080
- https-proxy                           : http://corporate.proxy:8080
```

## Host Operating System
```bash
ProductName:	macOS
ProductVersion:	12.5
BuildVersion:	21G72
```

### Steps to reproduce

  1. crc start
### Expected
Nice running OpenShift Local instance.
Note: laptop is using corporate proxy, therefore there are starting issues.

### Logs
[logs from crc start --log-lever debug](https://github.com/bandrzejewski/logsOSL/blob/main/OSLbootLogs.txt)

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
This looks like duplication of #3328 can you provide more info around proxy?

**Comment by bandrzejewski** [comment]
> This looks like duplication of #3328 can you provide more info around proxy?

What details do you need?

I saw the #3328 ticket, it's quite similar, but without resolution - so I thought I ll ask around.

**Comment by praveenkumar** [comment]
@bandrzejewski we need your proxy configuration details, if you are using the certs and allow to share then please do. Also https://github.com/code-ready/crc/issues/3328#issuecomment-1231902921 if you can provide it.

**Comment by praveenkumar** [comment]
@bandrzejewski Thanks for the issue, we have some proxy fixes which is coming up in 2.26.0 release, do try and create one if that doesn't work.

/close
